google/cloud/bigquery/job/query.py

```diff
@@ -858,6 +910,14 @@ def priority(self):
         """
         return self.configuration.priority
 
+    @property
+    def search_stats(self):
```

Comment:
This needs typing, and docstring should likely some variation of "returns" vs "generates" when describing the property.